formats/detailed_guide/publisher/links.json

```diff
@@ -9,6 +9,12 @@
     "related_guides": {
       "$ref": "#/definitions/guid_list"
     },
+    "related_policies": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "policy_areas": {
```

Comment:
`policy_areas` is already part of the allowed links for every format, so it's not necessary to add this here.

https://github.com/alphagov/govuk-content-schemas/blob/master/formats/base_links.json#L31-L34